src/CommandLineParser.ts

```diff
@@ -70,26 +70,43 @@ export const argv: any = yargs.options({
         default: false,
         demandOption: false
     },
+    debug: {
+        alias: 'd',
+        describe: 'Set logging level to debug (only use this if you know what are doing)',
+        type: 'boolean',
+        default: false,
+        demandOption: false
+    },
+    selectQuality: {
+        alias: 'q',
+        describe: 'Select the quality with a number 1 (worst) trough 10 (best), 0 prompt the user for each video',
+        default: 10,
+        type: 'number',
+        demandOption: false
+    },
     closedCaptions: {
         alias: 'cc',
         describe: 'Check if closed captions are available and let the user choose which one to download (will not ask if only one available)',
         type: 'boolean',
         default: false,
         demandOption: false
     },
+    // NOTE: useless
```

Comment:
We should remove all parameters that no longer make sense in the aria2c game.

_A software project is not finished when you add more code, but when there's nothing left to remove._

Reply:
done